VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participant-display-name/private-consultation-participant-display-name.component.ts

```diff
@@ -0,0 +1,19 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+@Component({
+    selector: 'app-private-consultation-participant-display-name',
+    templateUrl: './private-consultation-participant-display-name.component.html',
+    styleUrls: ['./private-consultation-participant-display-name.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PrivateConsultationParticipantDisplayNameComponent {
+    @Input() displayName: string;
+    @Input() isAvailable: boolean;
+    @Input() isInCurrentRoom: boolean;
+
+    constructor() {}
+
+    getColor(): string {
+        return this.isInCurrentRoom ? 'yellow' : this.isAvailable ? 'white' : '';
```

Comment:
Sonarcloud doesn't like nested ternary statements so look at another way to do this.
https://sonarcloud.io/project/issues?id=vh-video-web&issues=AX53Sxt-oS4oDdGZjJVA&open=AX53Sxt-oS4oDdGZjJVA&pullRequest=1766

Reply:
Updated